src/components/details/TabsPanelContainer.tsx

```diff
@@ -1,10 +1,9 @@
-import Tab from "@mui/material/Tab";
 import Box from "@mui/material/Box";
 import { padding } from "../../styles/constants";
 import StyledTabs from "../common/tab/StyledTabs";
 import StyledTab from "../common/tab/StyledTab";
 import { useLocation } from "react-router-dom";
-import { FC, SyntheticEvent, useEffect, useState } from "react";
+import React, { SyntheticEvent, useEffect, useState } from "react";
```

Comment:
Not related to your change, but there is a function from Lyn call allyProps, can change it to const allyProps = (index:number) => ....   which align with other function signature?